ar_exercise/exercise.rb

```diff
@@ -23,26 +23,41 @@ class Apartment < ActiveRecord::Base
 ################################################
 
 
-################################################
+# ###############################################
 # FINDING / SELECTING
-################################################
+# ###############################################
 
 # Hint, the following methods will help: `where`, `all`, `find`, `find_by`
 
 # Use Active record to do the following, and store the results **in a variable**
 # example: get every tenant in the DB
 all_tenants = Tenant.all
-
+binding.pry
 # get the first tenant in the DB
+first_tenant = Tenant.first
+binding.pry
 # get all tenants older than 65
+elders = Tenant.where("age > ?", "65")
+
 # get all apartments whose price is greater than $2300
+expensive = Tenant.where("monthly_rent > ?", "2300")
 # get the apartment with the address "6005 Damien Corners"
+damien_corners = Apartment.find_by(address: "6005 Damien Corners")
 # get all tenants in that apartment
-
+damien_tenants = Tenant.all.where(apartment: damien_corners)
 # Use `each` and `puts` to:
 # Display the name and ID # of every tenant
+all_tenants.each do |x|
+  puts x[:name] + " " + x[:id].to_s
+end
 # Iterate over each apartment, for each apartment, display it's address and rent price
+Apartment.all.each do |x|
+  puts x[:address] + " " + x[:monthly_rent].to_s
+end
 # Iterate over each apartment, for each apartment, display it's address and all of it's tenants
+# Apartment.all.each do |x|
+#   puts x[:address] + " " + x[:tenants].to_s
+# end
```

Comment:
in the same way we got all of damien_corners tenants we can get each individual apartments:

```ruby
Apartment.all.each do |apartment|
  # some code to display address
  apartment.tenants.each do |tenant|
    # code to display tenant
  end
end
```